From: Blocked Browser Team <dev@blockd.com>
Date: Mon, 24 Nov 2025 13:00:00 +0000
Subject: [PATCH 3/3] Add Blocked browser branding

Change product name, user agent, and icons to Blocked branding.

diff --git a/chrome/app/chromium_strings.grd b/chrome/app/chromium_strings.grd
index 5678901..efghijk 100644
--- a/chrome/app/chromium_strings.grd
+++ b/chrome/app/chromium_strings.grd
@@ -10,7 +10,11 @@
   <translations>
   </translations>
   <release seq="1">
+#ifdef BLOCKED_BROWSER
+    <message name="IDS_PRODUCT_NAME" desc="The product name">
+      Blocked Interview Browser
+    </message>
+#else
     <message name="IDS_PRODUCT_NAME" desc="The product name">
       Chromium
     </message>
+#endif
     <message name="IDS_SHORT_PRODUCT_NAME" desc="The short product name">

diff --git a/content/common/user_agent.cc b/content/common/user_agent.cc
index 6789012..fghijkl 100644
--- a/content/common/user_agent.cc
+++ b/content/common/user_agent.cc
@@ -50,6 +50,10 @@ std::string GetUserAgent() {
   std::string user_agent;
   user_agent = "Mozilla/5.0 (" + os_info + ") ";
   user_agent += "AppleWebKit/537.36 (KHTML, like Gecko) ";
+#ifdef BLOCKED_BROWSER
+  user_agent += "BlockedBrowser/1.0.0 ";
+  user_agent += "Chrome/" + version + " ";
+#else
   user_agent += "Chrome/" + version + " ";
+#endif
   user_agent += "Safari/537.36";

   return user_agent;
